update-cache/lib/UpdateCacheUrlProvider.js

```diff
@@ -40,7 +40,7 @@ class UpdateCacheUrlProvider {
    * @returns {Array<Object>} an array with objects containing the cache ID, cache name and
    * update-cache url.
    */
-  calculateFromOriginUrl(originUrl, timestamp) {
+  calculateFromOriginUrl(originUrl, timestamp = defaultTimestamp_()) {
```

Comment:
Can you also update the JS DOC to mark it as optional.

Reply:
done.